### DISCUSSION TITLE
PoPageDynamicForms propriedade p-multiple no PoLookup

### DISCUSSION THREAD

**Discussion by tiagodefariasantos** [po-ui/po-angular#1216] [Tire suas dúvidas]
Há a possiblidade de criar um PoLookup via PoPageDynamicForms que permita múltiplas seleção?
Olhando a documentação do PoLookup hoje é possível essa implementação, porém não consegui identificar o mesmo comportamento no PoDynamicForms

**Comment by jnrpalma** [comment]
Olá boa tarde @tiagodefariasantos.
Nesse momento não temos essa funcionalidade, dessa forma caso queira pode abrir uma issue de melhoria.
